### ISSUE TITLE
Website: Left nav is innaccessible by keyboard on component pages

### ISSUE THREAD

**Issue by Jahnp** [OfficeDev/office-ui-fabric-react#6955]
### Environment Information 
- __Package version(s)__: OUIFR@6.96.0; @uifabric/fabric-website@6.7.1
- __Browser and OS versions__: All

### Please provide a reproduction of the bug ~~in a codepen~~ on the livesite:
 Go to any component page on the website (e.g. [Button](https://developer.microsoft.com/en-us/fabric#/components/button)). Attempt to tab to the left nav to filter or visit another page.

#### Actual behavior:
The left nav cannot be reached by tab navigation. You have to use page search (e.g. CTRL/CMD + 'f') to apply focus to a link in the left nav.
 
#### Expected behavior:
The left nav can be reached by tab like on the Styles pages, e.g. [Animations](https://developer.microsoft.com/en-us/fabric#/styles/animations).

#### Priorities and help requested:
Are you willing to submit a PR to fix? Yes! See #6958
 
Requested priority: Normal
 
Products/sites affected: uifabric.io
